llama_index/prompts/default_prompts.py

```diff
@@ -318,3 +318,21 @@
 )
 
 DEFAULT_PANDAS_PROMPT = Prompt(DEFAULT_PANDAS_TMPL, prompt_type=PromptType.PANDAS)
+
+
+############################################
+# JSON Path
+############################################
+
+DEFAULT_JSON_PATH_TMPL = (
+    "We have provided a JSON schema below:\n"
+    "{schema}\n"
+    "Given a task, respond with a JSON Path query that "
+    "can retrieve data from a JSON value that matches the schema.\n"
+    "Task: {query_str}\n"
+    "JSONPath: "
```

Comment:
I'm surprised that this works without providing examples of what JSONPath query looks like.
Saw the notebook that you tried with GPT4. Does it work with weaker models?

Reply:
I think might be worthwhile to do a bit of prompt engineering here, and ideally have this work for at least text-davinci and turbo as well.

Reply:
So I tried with `text-davinci-003` & `gpt-3.5-turbo` and they seem to do just as well on the example JSON in the notebook. I think with this approach, it'll be very important for users to fill out the correct JSON schema *with* description fields. I'll include a note about that in the example Jupyter notebook